env/default.json.example

```diff
@@ -22,7 +22,7 @@
   "nodeTypes": { "MAINNET": "mainnet", "TESTNET": "testnet", "CUSTOM": "custom" },
   "networks": {
     "mainnet": {
-      "nodes": []
+      "nodes": ["http://api.experimental.symboldev.network:3000"]
```

Comment:
do we need this here?

Reply:
It is only for testing both networks. Should we remove it for release?